examples/generative/corrdiff/conf/config_generate.yaml

```diff
@@ -27,7 +27,7 @@ reg_ckpt_filename: "./checkpoints/regression.mdlus"
 image_outdir: "image_outdir"
   # Where to save the output images
 seeds: "0-63"
-  # Random seeds used for generation
+  # Number of samples on the ensemble dimension
```

Comment:
This is technically not the number of samples, but the random seeds that are used for each ensemble

Reply:
@mnabian 
if the user wants to run a generation with 100 members, that parameter controls that number right? there is nowhere else one can control the number of members.

Reply:
That is correct